alarm/PolicyManager2.js

```diff
@@ -1181,9 +1181,15 @@ class PolicyManager2 {
       }
     } finally {
       const action = policy.action || "block";
-      if (action === "block" || action === "app_block")
+      if (action === "block" || action === "app_block"){
         this.scheduleRefreshConnmark();
-    }
+      }else if (action === "route" ){
+        sem.sendEventToFireMain({
+          type: Message.MSG_OSI_UPDATE_NOW,
```

Comment:
Need to add some delayed execution on the event handler to prevent calling updateOSIPool repeatedly in a short time.